lib/inky/components/base.rb

```diff
@@ -0,0 +1,42 @@
+module Inky
+  module Components
+    class Base
+      tags = %w[class id href size large no-expander small target]
+      tags = tags.to_set if tags.respond_to? :to_set
+
+      IGNORED_ON_PASSTHROUGH = tags.freeze
+
+      attr_accessor :inky
+
+      def initialize(inky)
+        @inky = inky
+      end
+
+      def _pass_through_attributes(elem)
```

Comment:
I'm not sure why these methods started with `_` (simili-private?) but if we are to move them we probably should use the standard conventions and remove the leading `_`. You can make them `protected` if you like, but I don't feel that would be necessary.